model_compression_toolkit/keras/quantizer/mixed_precision/selective_quantize_config.py

```diff
@@ -0,0 +1,217 @@
+# Copyright 2021 Sony Semiconductors Israel, Inc. All rights reserved.
+#
```

Comment:
year :)

Reply:
Just the file name was changed, it is not a new file